modules/ui_parameters.py

```diff
@@ -46,8 +46,8 @@ def create_ui(default_preset):
                                 shared.gradio['dry_sequence_breakers'] = gr.Textbox(value=generate_params['dry_sequence_breakers'], label='dry_sequence_breakers', info='Tokens across which sequence matching is not continued. Specified as a comma-separated list of quoted strings.')
 
                             with gr.Blocks():
-                                shared.gradio['xtc_threshold'] = gr.Slider(0, 0.5, value=generate_params['xtc_threshold'], step=0.01, label='xtc_threshold', info='If there are multiple tokens with predicted probability at least xtc_threshold...')
-                                shared.gradio['xtc_probability'] = gr.Slider(0, 1, value=generate_params['xtc_probability'], step=0.01, label='xtc_probability', info='...remove all except the least probable one from sampling, with probability xtc_probability.')
+                                shared.gradio['xtc_threshold'] = gr.Slider(0, 0.5, value=generate_params['xtc_threshold'], step=0.01, label='xtc_threshold', info='If 2 or more tokens have probability above this threshold, consider removing all but the last one.')
```

Comment:
@oobabooga 

I find these descriptions a lot harder to understand now. What does "consider removing" mean? This isn't clear at first glance. Also "last" is not obviously a synonym for "least probable".

Reply:
I'm opposite. His descriptions helped.

Reply:
This is a matter of personal choice; as @Ph0rk0z, I find the new description easier to understand, although I agree it's not perfect.